### DISCUSSION TITLE
Compile process of iperf3 for windows 10

### DISCUSSION THREAD

**Discussion by Hariprasad97** [esnet/iperf#1791] [Q&A]
How could I compile iperf3 source code for windows10??

**Comment by davidBar-On** [comment]
See [these instructions](https://www.embeddedsystemtesting.com/2014/08/how-to-compile-iperf3-for-windows.html) which are also referenced from this [BudMan site](https://www.neowin.net/forum/topic/1234695-iperf-39-windows-build/#comments).
